regtests/README.md

```diff
@@ -85,6 +85,15 @@ project, just run:
 env POLARIS_HOST=localhost ./regtests/run.sh
 ```
 
+The catalog federation tests rely on the following configurations in `application.properties` to
+be set in order to succeed.
+
+```
+polaris.features."SUPPORTED_CATALOG_STORAGE_TYPES"=["S3","GCS","AZURE", "FILE"]
```

Comment:
super nit:
```suggestion
polaris.features."SUPPORTED_CATALOG_STORAGE_TYPES"=["S3","GCS","AZURE","FILE"]
```

Reply:
Do we not also rely on `readiness.ignore-severe-issues`?

Reply:
And `ALLOW_INSECURE_STORAGE_TYPES`

Reply:
I just realized that we don't need the "FILE" parameter either because it's included in the JVM args. I've removed it.